src/ODINN.jl

```diff
@@ -70,6 +70,7 @@ const global root_plots::String = joinpath(root_dir, "plots")
 include(joinpath(root_dir, "src/setup/config.jl"))
 # Losses
 include(joinpath(root_dir, "src/losses/Losses.jl"))
+include(joinpath(root_dir, "src/losses/Regularizations.jl"))
```

Comment:
This is what makes the tests fail
```suggestion
include(joinpath(root_dir, "src/losses/Regularization.jl"))
```